examples/tutorials/introduction/4_properties.py

```diff
@@ -0,0 +1,207 @@
+"""
+Property Traits
+===============
+
+The ``Image`` class has three traits which are closely related: ``scan_size``,
+``scan_width`` and ``scan_height``.  We would ideally like to keep all of these
+synchronized.  This can be done with trait observation, as shown in the previous
+section, but this sort of pattern is common enough that Traits has some in-built
+helpers.
+
+Instead of declaring that ``scan_width = Float()`` we could instead declare it
+to be a ``Property`` trait type.  Property traits are similar to ``@property``
+decorators in standard Python in that rather than storing a value, they compute
+a derived value via a "getter", and optionally store values via a "setter".
+Traits uses specially named methods of the form ``_get_<property>`` and
+``_set_property`` for these "getters" and "setters."  If there is a "getter"
+but no "setter" then the property is read-only.
+
+Additionally, we need to know when the value of the property might change, and
+so we need to declare what traits to observe to know when the property might
+change.  What all this this means is that we can define ``scan_width`` as a
```

Comment:
Overlooked this earlier:

```suggestion
change.  What all this means is that we can define ``scan_width`` as a
```